torch/distributed/tensor/parallel/_utils.py

```diff
@@ -151,6 +151,6 @@ def _create_1d_device_mesh(device_mesh: DeviceMesh, tp_mesh_dim: int = 0) -> Dev
         if cur_rank in mesh_1d:
             res_sub_mesh = sub_mesh
 
-    sub_pg = device_mesh.get_dim_groups()[tp_mesh_dim]
-    res_sub_mesh._dim_groups = [sub_pg]
+    res_sub_mesh._dim_group_ranks = [device_mesh._dim_group_ranks[tp_mesh_dim]]
+    res_sub_mesh._dim_group_tags = [device_mesh._dim_group_tags[tp_mesh_dim]]
```

Comment:
we can combine them into one line with the suggestion on top, right? I think that would be better, what do you think?